sys-libs/vulkan/vulkan-1.4.310.recipe

```diff
@@ -73,7 +73,7 @@ BUILD()
 		-DBUILD_WSI_XCB_SUPPORT=OFF \
 		-DBUILD_WSI_XLIB_SUPPORT=OFF \
 		-DBUILD_WSI_WAYLAND_SUPPORT=ON \
-		-DVulkanHeaders_DIR=$sourceDir2/build/CMakeFiles/Export/2305fcc30b4aca20c8b0c09c18da2b50/ \
+		-DVulkanHeaders_DIR=$sourceDir2/build/CMakeFiles/Export/2305fcc30b4aca20c8b0c09c18da2b50 \
```

Comment:
In line 77 those paths should be defined with finddir
Also $libDir/cmake should be moved to the _devel package.